sdk/eventhub/azure-eventhub/pyproject.toml

```diff
@@ -1,3 +1,46 @@
+[build-system]
+requires = ["setuptools>=77.0.3", "wheel"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "azure-eventhub"
+authors = [
+    {name = "Microsoft Corporation", email = "azpysdkhelp@microsoft.com"},
+]
+description = "Microsoft Azure Event Hubs Client Library for Python"
+keywords = ["azure", "azure sdk"]
+requires-python = ">=3.9"
+license = "MIT"
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3 :: Only",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+    "Programming Language :: Python :: 3.13",
+]
+dependencies = [
+    "azure-core>=1.27.0",
+    "typing-extensions>=4.0.1",
+]
+dynamic = ["version", "readme"]
+
+[project.urls]
```

Comment:
do we also want to link our other docs such as changelog etc https://packaging.python.org/en/latest/guides/writing-pyproject-toml/#urls

Reply:
ohh interesting. I think this is something we can add in the future? As of right now, this follows the pyproject template we're using in code gen.